configs/royaloak-ctm.board

```diff
@@ -102,7 +102,7 @@ pins = (
     ('uart2_rts_pg2',        'rsvd2',    None,      'down', True,  False, False, False),
     ('uart2_cts_pg3',        'rsvd2',    None,      'down', True,  False, False, False),
     ('uart1_tx_pu0',         'uarta',    None,      'none', False, False, False, False),
-    ('uart1_rx_pu1',         'uarta',    None,      'none', False, True,  False, False),
+    ('uart1_rx_pu1',         'uarta',    None,      'down', False, True,  False, False),
```

Comment:
Is that correct that it's uart1 here but in the csv it's uart2?

Reply:
Thank you doug, i didn't catch that. Your right